.github/workflows/push-nonroot.yml

```diff
@@ -0,0 +1,30 @@
+name: Build & Push Nonroot Container Image
+on:
+  push:
+    branches:
+    - master
+    tags:
+    - v*
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      DOCKER_BUILDKIT: "1"
+      IMAGE: docker.io/${{ github.repository }}
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set version latest
+      if: github.ref == 'refs/heads/master'
+      run: echo VERSION=latest-nonroot >> $GITHUB_ENV
+    - name: Set version from tag
+      if: startsWith(github.ref, 'refs/tags/v')
+      run: echo VERSION=$(echo ${GITHUB_REF#refs/tags/})-nonroot >> $GITHUB_ENV
+    - name: Build Image
+      run: docker build -t "${IMAGE}:${VERSION}" .
```

Comment:
The tag contains the suffix `-nonroot` but the build does not use the any target. Doesn't this result in an image runnig as root?